crates/red_knot_python_semantic/resources/mdtest/binary/booleans.md

```diff
@@ -0,0 +1,25 @@
+## Binary operations on booleans
+
+## Basic Arithmetic
+
+```py
+a = True
+b = True
```

Comment:
Especially if we go for a bit more precision, as I suggest below, it would be useful to also test with `True` and `False`, or `False` and `False`. We don't need the full matrix, but at least some different variants for different operators. And maybe all four possible combinations for `|`, since that one is special for bools.

Reply:
I added more tests covering those cases.